src/main/java/io/cdap/plugin/gcp/spanner/source/SpannerSourceConfig.java

```diff
@@ -24,12 +25,16 @@
 import io.cdap.plugin.gcp.spanner.common.SpannerUtil;
 
 import java.io.IOException;
+import java.util.Set;
 import javax.annotation.Nullable;
 
 /**
  * Spanner source config
  */
 public class SpannerSourceConfig extends GCPReferenceSourceConfig {
+  // todo CDAP-14233 - add support for array
+  private static final Set<Schema.Type> SUPPORTED_TYPES =
```

Comment:
can't this be re-used with SpannerSinkConfig?

Reply:
This is because both of them support different types. Sources does not allow importing int and float.

Reply:
why is that?